### ISSUE TITLE
"stack ghci" should either use haskell2010, or pay attention to the cabal file language specification

### ISSUE THREAD

**Issue by mgsloan** [commercialhaskell/stack#2092]
From nitrix on #haskell, line 15 of http://lpaste.net/162186 has a `do` block with the contents aligned with the `let`.  It builds with haskell98 but not haskell2010.

**Comment by nitrix** [comment]
Actually, it compiles with `stack ghci` but not with `stack build`.
The error says `Parse error on a binding <- of a do block.`
The project is indeed haskell2010; we suspect it could be the issue.

**Comment by mgsloan** [comment]
Right, `stack ghci` is running in haskel98 mode. `stack build` knows to use haskell2010.
